src/merge_tools.rs

```diff
@@ -233,32 +229,27 @@ pub fn run_mergetool(
         .try_collect()?;
 
     let args = interpolate_mergetool_filename_patterns(&editor.merge_args, &paths);
-    let args_str = args
-        .iter()
-        .map(|p| p.to_string_lossy().into_owned())
-        .collect_vec();
-    let exit_status = Command::new(&editor.program)
-        .args(args)
-        .status()
-        .map_err(|e| ExternalToolError::FailedToExecute {
-            tool_binary: editor.program.clone(),
-            args: args_str.clone(),
-            source: e,
-        })?;
+    let mut cmd = Command::new(&editor.program);
+    let exit_status = cmd.args(args).status();
+    // TODO: Have a --debug flag for debugging information (see other uses of
+    //    tracing::debug) a separate --verbose flag for printing this kind of
+    //    information.
```

Comment:
No idea if we should add more detailed filtering options. We can use `RUST_LOG=` if `-v` wasn't enough.

https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html

Reply:
There's a discussion about this on Discord. You could chime in.

Currently, most of the extra information `-v` prints really seems to be debugging information. Here, the information is more user-facing. It might make sense to replace what -v does now with an environment variable. Or maybe it's not very important. I can't decide if I care.

Reply:
If I understand it, we prefer using `-v` for this kind of information, not for controlling status/log/diff/show verbosity.

Reply:
I'll delete the TODO.

Reply:
Maybe some of the current tracing events should even be at the `TRACE` level, or maybe this should be at INFO level, I don't know. We can adjust them up or down later if it feels like we don't use the levels right.